src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalRent = 40 * days;
+
+  if (days >= 7) {
+    return totalRent - 50;
+  } else if (days >= 3) {
```

Comment:
you don't need else here, your conditions do not eliminate each other
```suggestion
  }

if (days >= 3) {
```